src/component/Timer/index.jsx

```diff
@@ -22,11 +31,43 @@ const Timer = () => {
 
     const reset = () => {
       timeOverSoundAudio.play();
+      const showNotification = () => {
+        let text = null;
+        const img = 'https://source.unsplash.com/1600x900/?nature,water;';
+        if (counter.current === 6) {
+          text = `Great job! Take a long break. You have ${PERIODS.longBrk.secs} minutes.`;
+          (() => new Notification('Pomodoro', {
+            body: text,
+            icon: img
+          }))();
+        } else if (currentPeriod.id === 'short-brk' || currentPeriod.id === 'long-brk') {
+          text = `Time to get back to work! Your next break starts in ${PERIODS.work.secs} minutes.`;
+          (() => new Notification('Pomodoro', {
+            body: text,
+            icon: img
+          }))();
+        } else if (currentPeriod.id === 'work') {
+          text = `Nice work! Take a short break. You have ${PERIODS.shortBrk.secs} minutes.`;
+          (() => new Notification('Pomodoro', {
+            body: text,
+            icon: img
+          }))();
```

Comment:
Why are you repeating this line of code 3 times? The whole point of defining `text` variable is so you don't repeat this line more than once. Can you figure out how to do that?

Reply:
yes, I knew there is a problem if I repeat myself with the same code, but I didn't think about it a lot. I will think about it now and try to figure out it.